src/main/scala/com/microsoft/hyperspace/index/IndexConstants.scala

```diff
@@ -57,4 +57,7 @@ object IndexConstants {
 
   val REFRESH_DELETE_ENABLED = "spark.hyperspace.index.refresh.delete.enabled"
   val REFRESH_DELETE_ENABLED_DEFAULT = "false"
+
+  val REFRESH_LOGENTRY_ACTION_ENABLED = "spark.hyperspace.index.refresh.logentry.enabled"
```

Comment:
@pirz Could you please explain what this config corresponds to?

Also, the PR description has a different config `spark.hyperspace.index.enforce.delete.on.read.enabled`. Could you update as appropriate? Thank you!

Reply:
We have only one config and that is: "spark.hyperspace.index.refresh.source.content.enabled".
PR description is updated to reflect latest changes.